compiler/src/scala/quoted/runtime/impl/QuoteMatcher.scala

```diff
@@ -463,24 +460,32 @@ object QuoteMatcher {
             // No Match
             case _ =>
               notMatched
+      end runMatch
+
+      if (ctx.settings.YdebugMacros.value)
+        try {
+          runMatch()
+        } catch {
+          case e: RuntimeException =>
```

Comment:
Just to make sure we only print this when the match fails due to a match error. Otherwise, we would get the stack trace.
```suggestion
          case e: util.boundary.Break[?] =>
```

Reply:
That's a good catch. Thank you.